### PR TITLE
Updates file validator test

### LINKED ISSUE #112: G5.112 Update automated tests in ValidDataFile to create files using the Test Trait
### Branch

g5.112-updateValidDataFileTests

### Groups

Group 5 | Automated Testing | Performance

### Describe

The tests for the validator `ValidDataFile` ([`trpcultivate_phenotypes/tests/src/Kernel/Validators/ValidatorValidDataFileTest.php`](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/blob/da06095cc44fa62e4e7698bb2e8ca94c94bd94cc/trpcultivate_phenotypes/tests/src/Kernel/Services/Traits/ValidTraitTest.php)) need to be updated to use the [`createTestFile()`](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/blob/da06095cc44fa62e4e7698bb2e8ca94c94bd94cc/trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php#L181) method in the `PhenotypeImporterTestTrait` class to create test files. 

### Design

See below for an example where this is used within this module:
https://github.com/TripalCultivate/TripalCultivate-Phenotypes/blob/da06095cc44fa62e4e7698bb2e8ca94c94bd94cc/trpcultivate_phenotypes/tests/src/Kernel/TripalImporter/TraitImporterFormValidateTest.php#L238-L242

### PR DESCRIPTION
**Pull request by reynoldtan**
**Issue #112 Update automated test in ValidDataFile**

## Motivation
This PR will update automated test in ValidDataFile.

## What does this PR do?
*Please describe each things this PR does. For example, a PR may 1) solve a specific bug, 2) create an auomated test to ensure it doesn't return.*

1. Implement Phenotype Trait method that will create a test file.

### Automated Testing
*Please describe each automated test this PR creates and provide a list of the assertions it makes using casual language.*
*Do not just say things like "asserts the array is not empty" but rather say "Ensures that the return value of method X with these parameters is not an empty array".*

ClassName->testSpecificBug: tests this specific bug
- 
- 
- 

### Manual Testing
*Describe in detail how someone should manually test this functionality.*
*Make sure to include whether they need to build a docker from scratch, create any records, etc.*

### CODE REVIEW

**Comment by carolyncaron** [comment]
**Status**: This PR is currently waiting on PR #108 to be merged before it can be reviewed.

File: trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php [hunk 1]
```diff
@@ -183,69 +183,77 @@ protected function createTestFile($details) {
...
-    }
-    else if (array_key_exists('file', $details['content'])) {
-      $size = @filesize($full_path);
-      // Set size unless there was an error.
-      $this->assertNotFalse($size, 'Unable to determine size of test file: ' . $full_path);
-      $file->setSize($size);
-    }
+    // Reference file attributes:
+    $file_id = $file->id();
+    $file_uri = $file->getFileUri();
```
Comment (carolyncaron) [inline trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php:205]: Just out of curiosity, why does $file_uri need to be redefined here after it was defined on line 193? Shouldn't it still be the same or is it possible that it changed?

**Review by carolyncaron — APPROVED** [review]
This makes sense to me although I think it would be ultimately helpful to have @laceysanderson take a look at it, who is more familiar with the Test Trait.

File: trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php (hunk 1 above)
Comment (reynoldtan) [reply trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php:205]: They could be the same, I am not sure of how it's stored, but the 205 clearly implies that all subsequent modifications to the file will refer to the file uri, id etc. of the file object that it just created.

**Review by laceysanderson — COMMENTED** [review]
This does look all good to me as well :-)

**Pull request merged**